README.md

```diff
@@ -121,6 +125,57 @@ Note: This is not filled out yet! ...
 ### Usage
 Note: This is not filled out yet! ...
 
+## :framed_picture: Icons Set
+
```

Comment:
Please maintain consistency without the new lines after the headings

Reply:
@angsherpa456 there is always a new line after a '##' heading